src/main/java/wolox/training/models/Book

```diff
@@ -0,0 +1,128 @@
+package wolox.training.models;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column(nullable = false)
+    private String author;
+
+    @Column(nullable = true)
+    private String genre;
+
+    @Column(nullable = false)
+    private String image;
+
+    @Column(nullable = false)
+    private String subtitle;
+
+    @Column(nullable = false)
+    private String year;
+
+    @Column(nullable = false)
+    private String publisher;
+
+    @Column(nullable = false)
+    private String isbn;
+
+    @Column(nullable = false)
+    private int pages;
+
+    public Book(String bookTitle, String bookAuthor, String bookGenre, String bookImage, String bookSubtitle, String bookYear, String bookPublisher, String bookIsbn, int bookPages) {
```

Comment:
You need to also add the default constructor 
```
public Book(){}
```
JPA needs it to instantiate objects when bringing them from the DB.

Reply:
ok, this default constructor is supposed to be empty?

Reply:
Yes